src/Package/Test/Options/EncodingsTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.R.Host.Client;
+using Microsoft.UnitTests.Core.XUnit;
+using Microsoft.VisualStudio.R.Package.Options.R.Tools;
+using Microsoft.VisualStudio.R.Package.Test.Utility;
+using Xunit;
+using static System.FormattableString;
+
+namespace Microsoft.VisualStudio.R.Package.Test.Options {
+    [ExcludeFromCodeCoverage]
+    [Collection(CollectionNames.NonParallel)]
+    public class EncodingsTest {
+        [Test]
+        public async Task ValidateEncodings() {
```

Comment:
Perhaps make it `[CompositeTest]` with one test per page? So that when it fails, we know right away which one.